graphql.go

```diff
@@ -63,16 +63,24 @@ type Schema struct {
 	schema *schema.Schema
 	res    *resolvable.Schema
 
-	maxDepth         int
-	maxParallelism   int
-	tracer           trace.Tracer
-	validationTracer trace.ValidationTracer
-	logger           log.Logger
+	maxDepth              int
+	maxParallelism        int
+	tracer                trace.Tracer
+	validationTracer      trace.ValidationTracer
+	logger                log.Logger
+	useStringDescriptions bool
 }
 
 // SchemaOpt is an option to pass to ParseSchema or MustParseSchema.
 type SchemaOpt func(*Schema)
 
+// UseStringDescriptions disables the parsing of comments as descriptions
```

Comment:
Can you make this comment a little bit more descriptive, please? This comment will be used by `godoc` to generate documentation for the library and some users might not even know about strings vs comments as descriptions.